apps/portal/src/app/features/file-processor/file-processor.component.ts

```diff
@@ -13,9 +14,26 @@ export class FileProcessorComponent implements OnInit {
   // to do:the name will probably be changed to match the backend
   jobs: JobDetails[] = [];
 
-  constructor(private http: HttpClient) {}
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  summaries: any[] = [];
+
+  constructor(
+    private http: HttpClient,
+    private fileService: FileService,
+  ) {}
 
   ngOnInit(): void {
+    const jobIds = [1, 2]; // Once file upload api is integrated will change this code
```

Comment:
We are supposed to get job ids from localstorage/sessionstorage(better sessionstorage).
How were you planning to fetch it?
As mentioned in the task, the job ids will be saved in sessionstorage when uploading file.
And summaries will fetcehd using that data.

Reply:
Right now, the file upload API is not integrated yet. I will be saving the job IDs in local storage in that task. Once we have that array in local storage, I will make the necessary changes here. For now, I have hardcoded it to [1, 2] after uploading two files using Postman so I can check whether the API is giving the correct response in the UI or not